src/main/java/org/catmaid/ScaleCATMAID.java

```diff
@@ -72,46 +72,55 @@
 	{
 		public int tileWidth;
 		public int tileHeight;
+		public long minC;
```

Comment:
It's not about accuracy but about being possible at all.  We cannot scale to the final level without having scaled everything beforehand.  One could certainly generate one new scale level at a time, do that in chunks in parallel and after that do the next scale level in chunks.  I think that's a good idea but not what's implemented here.  Do you want to do that?  As an alternative tool?

You are right about the mechanism to keep going despite missing tiles on the road.  It was convenient to assume that if there is no data we could stop because it works without knowing about the width and height of the dataset.  For that, however, we need only the physical width and height of the dataset, so only two parameters, conveniently in pixel coordinates at scale level 0 to not confuse stuff.  These parameters are only required if ignoreEmptyTiles is true.  What do you think?